mitodl/mit-open
# license_cc and continuing_ed_credits fields

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/5133
Closes https://github.com/mitodl/hq/issues/5132

### Description (What does it do?)
Adds `license_cc` (default `False`) and `continuing_ed_credits` (default`None`) fields to `LearningResource` and updates relevant ETL pipelines to populate those fields.


### How can this be tested?
Set `OCW_*`, `SEE_*` and `XPRO_*` env settings to the same as RC.

Run `docker compose up`.  The new migration should run and set `license_cc` to `True` for OCW resources.  Everything else will have the default `False`.

Run the following, when it's done, many of the ingested courses should have a non-null `continuing_ed_credits` numerical value.
```
./manage.py backpopulate_sloan_data
./manage.py backpopulate_xpro_data
```

Run the following, then check that the course has `license_cc=True`:
```
./manage.py backpopulate_ocw_data --course-name 14-44-energy-economics-spring-2007 --skip-contentfiles
```
